fusor-ember-cli/app/controllers/undercloud-deploy.js

```diff
@@ -104,6 +104,11 @@ export default Ember.Controller.extend(DeploymentControllerMixin, NeedsDeploymen
                 promise.then(fulfill);
                 console.log('create success');
                 console.log(response);
+                if (self.get('applicationController.isEmberCliMode')) {
+                  // only used for develoment to enabled OSP tabs (disableOspTab: false)
+                  model.set('openstack_undercloud_password', 'this-passwd-is-populated by fusor/server')
```

Comment:
This is for `isEmberCliMode` mode which will also be used for the UI demo mock. In production, `openstack_undercloud_password` is saved by `fusor/server`, so for development with mocks and UI demo, we need to add a dummy password here so that the tabs were become enabled.